.github/workflows/push_image_pr.yml

```diff
@@ -0,0 +1,69 @@
+name: Build and push PR image to quay.io
+on:
+  pull_request_target
+
+env:
+  REGISTRY_USER: netobserv+github_ci
+  REGISTRY_PASSWORD: ${{ secrets.QUAY_SECRET }}
+  REGISTRY: quay.io/netobserv
+  IMAGE: network-observability-console-plugin
+  ALLOWED_USERS: mariomac jotak jpinsonneau OlivierCazade eranra KalmanMeth ronensc
+
+jobs:
+  push-pr-image:
+    name: push PR image
+    runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+        go: ['1.17']
+    steps:
+      - name: install make
+        run: sudo apt-get install make
+      - name: set up go 1.x
+        uses: actions/setup-go@v2
+        with:
+          go-version: ${{ matrix.go }}
+      - name: validate user
+        id: validate_user
+        run: |
+          allowed=(${{ env.ALLOWED_USERS }})
```

Comment:
@jotak as you suggest, I think that using a label might be more "clean" also I suggest triggering this also if someone is adding a level to a PR (is this possible?) .... this will allow working on a PR and if needed just tag the pr with "push_image" and this will cause in this action to start creating images for the PR

Reply:
We also need to make sure that only specific users will be able to add the labels to the PR

Reply:
Ok, I've changed to use a label .. it's cleaner and shorter. So the new workflow is:
1. A PR is opened
2. Maintainer makes sure this PR is safe to test (no suspicion of attack / stealing secret)
3. He adds the "ok-to-test" label
4. That triggers a build and push image to quay
5. If the PR is modified, the "ok-to-test" label is automatically removed. Then GOTO 2.

Also, I've modified the shortlived image creation to stick with what we said on FLP previously.

Reply:
of course this is not a mandatory workflow, we can continue to work as today, it's only something to follow if tester / QE wants an easy way to get the image

Reply:
BTW, setting a label requires write permission on the repo, so it should be safe

Reply:
👍